inc/switchtec/switchtec.h

```diff
@@ -273,6 +274,8 @@ int switchtec_event_wait(struct switchtec_dev *dev, int timeout_ms);
 
 /*********** Generic Accessors ***********/
 
+extern int mrpc;
```

Comment:
I would suggest that this belongs in switchtec/errors.h

Reply:
Also it needs a much better name. 'mrpc_error_cmd' or something like that. Global variables should always be verbosely named.

Reply:
Will update.